pandas/core/arrays/categorical.py

```diff
@@ -634,7 +634,7 @@ def from_codes(cls, codes, categories=None, ordered=None, dtype=None):
 
         Parameters
         ----------
-        codes : array-like, integers
+        codes : array-like, int
```

Comment:
not sure about this one. IMO not so clear. maybe array-like[int] or possibly Sequence[int]. @datapythonista ?

Reply:
we use `array-like of int` (or `list of int`...)